go/chat/emojisource.go

```diff
@@ -165,21 +207,25 @@ func (s *DevConvEmojiSource) normalizeShortName(shortName string) string {
 	return strings.ReplaceAll(shortName, ":", "") // drop any colons from alias
 }
 
-func (s *DevConvEmojiSource) validateShortName(shortName string) error {
+func (s *DevConvEmojiSource) validateShortName(shortName string) *EmojiValidationError {
 	if s.IsStockEmoji(shortName) {
-		return errors.New("cannot use existing stock emoji short name")
+		return NewEmojiValidationErrorJustError(errors.New("cannot use existing stock emoji short name"))
+	}
+	if len(shortName) > maxShortNameLength {
+		return NewEmojiValidationErrorJustError(fmt.Errorf("short name is too long, must be less than %d",
+			maxShortNameLength))
 	}
-	if len(shortName) > maxShortNameLength || len(shortName) < minShortNameLength {
-		return fmt.Errorf("short name (length %d) not within bounds %d,%d",
-			len(shortName), minShortNameLength, maxShortNameLength)
+	if len(shortName) < minShortNameLength {
+		return NewEmojiValidationErrorJustError(fmt.Errorf("short name is too short, must be greater than %d",
```

Comment:
This is more clear from the beginning but might still be too long for 5s: 

![image](https://user-images.githubusercontent.com/255797/78934360-a5c73880-7a5f-11ea-927d-8e04d1d331cc.png)

Reply:
or maybe we could just make the error display two lines on mobile?